src/enclave/enclave_mem.c

```diff
@@ -343,6 +237,7 @@ void* enclave_mmap(
     if (((intptr_t)ret) >= 0)
     {
         int found_only_fresh_pages = 0;
+        int mprotect_ret;
```

Comment:
Is this placed just to satisfy the function signature for `sgxlkl_host_syscall_mprotect`?

Reply:
Yes. You need to provide pointer to something that can be used for the return value. prior to David's work here, the return value was ignored. So it is with this.